sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -126,4 +126,10 @@ object StaticSQLConf {
       .intConf
       .createWithDefault(1000)
 
+  val MAX_BROADCAST_EXCHANGE_THREADNUMBER =
+    buildStaticConf("spark.sql.broadcastExchange.maxThreadNumber")
+      .doc("MAX number of threads can hold by BroadcastExchangeExec.")
```

Comment:
Can you elaborate this in the doc about what this number controls? For instance, it controls the parallelism of fetching and broadcasting the table.